collateral/deployment/k8s-without-helm-reference.md

```diff
@@ -0,0 +1,53 @@
+# Deploying Sourcegraph to Kubernetes (k8s) without Helm
+
+## Introduction
+This material is intended to be used as a handout to customers working with the Customer Engineering team. The materials are meant to serve as an overview and resource to aid in deep diving into Sourcegraph's documentation and common deployment tasks.
+
+**Table of Contents:**
+
+- [Introduction](#introduction)
+- [Sourcegraph Setup and Deployment](#sourcegraph-setup-and-deployment)
+	- [Prerequisites](#prerequisites)
+  - [Create your cluster](#create-your-cluster)
+  - [Configure your manifest file](#configure-your-manifest-file)
+  - [Install Sourcegraph](#install-sourcegraph)
+- [Next Steps](#next-steps)
+
+## Sourcegraph Setup and Deployment
+
+### Prerequisites
+- Ensure you have your Sourcegraph Enterprise License
+- Access to deploy Kubernetes clusters
+  - Minimum Kubernetes version: `v1.19` and `kubectl` `v1.19`
+  - Support for Persistent Volumes (SSDs recommended)
+- Determine resource requirements to properly size your instance: [Resource Estimator](https://docs.sourcegraph.com/admin/deploy/resource_estimator)
+- Install relevant CLIs for your provider of choice:
+  - AWS: [`aws` cli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) and [`ejsctk`](https://eksctl.io/introduction/#installation)
+  - Azure: [`az` cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+  - GCP: [`gcloud` cli](https://cloud.google.com/sdk/gcloud)
+
+### Create your cluster
+- Follow the instructions for your provider of choice to create your Kubernetes (k8s) cluster: [Cloud installation instructions](https://docs.sourcegraph.com/admin/deploy/kubernetes#cloud-installation)
+  - Follow the recommendations for `node type`, `disk type`, and `minimum` and `maximum` nodes
+
+### Configure your manifest file
+Sourcegraph has a list of configurations available and recommended that need to be made to your manifest file to properly install Sourcegraph into your cluster. Please review the full configuration guide prior to proceeding: [Configure Sourcegraph with Kubernetes](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure)
+
+Get started by following these steps to fork the [`deploy-sourcegraph`](https://github.com/sourcegraph/deploy-sourcegraph) repository: [Getting started](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure#getting-started)
+
+- Configurations:
+  - [License Key](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure#add-license-key)
```

Comment:
```suggestion
 ```
Same reason as above.

Reply:
Going to wait on this based on my [question above about that response](https://github.com/sourcegraph/customer-training/pull/5#discussion_r883038670)

(more so a note to myself on why I left the convo open, haha)